definitions.py

```diff
@@ -44,5 +44,6 @@ class TargetType(StrEnum):
 
 class CardCategory(StrEnum):
```

Comment:
Why is it called `CardCategory` if it's used for things other than cards? Plus Colorless is a class, not another type of card.

Reply:
Yeah, I'll fix this. Can you show me the `PlayerClass` enum you're using? I'll copy that.

Reply:
What Class are Status and Curse cards? Should they be considered "Colorless" as well?

Reply:
Make a new ´ANY´ class because some cards will spawn random colorless cards and we dont want curses or statuses to be included as well.